feature/git/dffml_feature_git/feature/operations.py

```diff
@@ -177,6 +177,21 @@
             'commit': sha
             }
 
+@op(inputs={
+        'repo': git_repository
+        },
+    outputs={
+        'score': bandit_output
+        },
+)
+async def run_bandit(repo: Dict[str, str]):
+    bandit_op = await check_output(sys.executable, '-m', 'bandit', 
```

Comment:
Make sure to use `-r`

Reply:
When looking at `-f json` output. Return value should be `_totals` from `metrics`, add to that object you're returning the count of how many `issues` have a `issue_severity == "HIGH" and issue_confidence == "HIGH"`